bin/propolis-standalone/src/main.rs

```diff
@@ -887,6 +913,12 @@ fn main() -> anyhow::Result<()> {
 
     let (log, _log_async_guard) = build_log();
 
+    // Check that vmm and viona device version match what we expect
+    if let Err(e) = api_version_checks(&log) {
+        slog::error!(&log, "IO error during API version checks");
+        return Err(e.into());
+    }
```

Comment:
The error's already going to be outputted to stderr so I'd just skip the extra log here:
```suggestion
    api_version_checks(&log).context("API version checks")?;
```

Reply:
Done